### PR TITLE
(DOCSP-11370) Update concise content page

### PR DESCRIPTION
**Pull request by atsansone**
@andf-mongodb : [Staged](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11370/style-guide/writing/write-clear-concise-sentences-paragraphs.html)

---
[Builds without errors on `style-guide-v3`.](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5f1efcef2f9a5cfb5b51cba6)

### CODE REVIEW

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt [hunk 1]
```diff
@@ -4,23 +4,24 @@
...
 walls of text, but we shouldn't force them to do that. Use the
-following guidelines to help you write clear and concise sentences and
-paragraphs.
+following guidelines to clarify and condense sentences and paragraphs.
 
 .. _use-consistent-sentence-structure:
 
 Use a Consistent Sentence Structure
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------------------------------
```
Comment (andf-mongodb) [inline source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:24]: General rule of thumb: don't force users to backtrack when reading a sentence. If they have to gather necessary details at the end to understand the beginning, see if you can rewrite it.

Example: 

"Gathering the data necessary to parse dense content effectively is a task struggled with by many users"

Is basically about as backwards as you can get. ** shudders **

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt [hunk 2]
```diff
@@ -54,56 +54,127 @@ Use a Consistent Sentence Structure
...
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - The modal recommends upgrading to new software.
+     - The modal contains a recommendation that the user upgrade to new
+       software.
+
+   * - It would benefit the users if they had this information.
```
Comment (andf-mongodb) [inline source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:156]: Even better: "It benefits the users to have this information"

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt [hunk 3]
```diff
@@ -54,56 +54,127 @@ Use a Consistent Sentence Structure
...
-  - Are you using adverbs (modifiers ending in *-ly*)? If so, you can
-    remove most of them without changing the meaning.
-  - What about adjectives? If they aren't necessary to the meaning,
-    remove them.
-  - Can prepositional phrases be shortened?
-  - Are you using empty phrases that don't clarify the content?
+  - Are you using adverbs (modifiers ending in *-ly*)? You can remove
+    most adverbs without changing the meaning.
+  - Are you using adjectives? Remove them if they aren't necessary to
+    the meaning.
```
Comment (andf-mongodb) [inline source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:173]: Some features include adjectives as part of their name. Be careful that you are not removing an adjective that functions as a vital identifier for a feature.  "Resumable Initial Sync" cannot be called anything else, for example, 

Generally, Jira tickets will be careful to call this out, either in the Scope doc, of even as a feature branch or Jira component thing.

Further, adjectives serve to differentiate cases where a statement applies ... and where it expressly doesn't. With this one:

Indexes Build Simultaneously on "Data-Bearing Replica Set Members"

Removing any of the adjs will render the statement misleading at best, or untrue at worst. (differing behavior for primary + secondaries vs arbiters vs non-data-bearing vs standalone vs sharded cluster members)

General gist of this is spot on: No one cares that the Resumable Initial Sync showed up to work in a blue suit on a cloudy day. But we can't call it, itself, anything different than what it is.

File: source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt (hunk 2 above)
Comment (atsansone) [reply source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt:156]: Did this look familiar?

**Pull request merged**